src/gluonts/mx/distribution/lds.py

```diff
@@ -229,6 +229,13 @@ def log_prob(
         # TODO: Based on form of the prior decide to do either filtering
         #   or residual-sum-of-squares
         log_p, final_mean, final_cov = self.kalman_filter(x, observed)
+        if scale is not None:
+            F = self.F
+            # log_abs_det_jac: sum over all output dimensions.
+            ladj = -F.sum(F.log(F.abs(scale)), axis=-1, keepdims=True)
```

Comment:
The fact that we simply sum over the output dimensions implies that the scaling is diagonal, right?

I would also ask you to double check the minus here (this agrees with `TransformedDistribution`, and you've probably done this so many times, but just to be sure)

Reply:
Yes (about scale being diagonal), in the preceding statement the target is divided by scales independently. The scaling transformation and the Jacobian adjustment are done locally there.

About the minus sign: yes, I double-checked. I always imagine this simple example of uniform distribution in [0, 100] where the log-density is `- log 100` and if you divide by 100, the transformed log-density is 0. So we need to subtract `log 100` to get the original.

Reply:
That's a good trick to remember it!